test_scripts/Defects/5_1/3196_SDL_responds_resultCode_GENERIC_ERROR_while_dataType_VEHICLEDATA_EXTERNTEMP_is_VEHICLE_DATA_NOT_AVAILABLE_and_not_in_subscribed_list_store.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/3196
+--
+-- Description:
+-- SDL responds "resultCode: SUCCESS" while dataType:VEHICLEDATA_EXTERNTEMP is VEHICLE_DATA_NOT_AVAILABLE and not
+-- in-subscribed list store
+-- Precondition:
+-- 1) SDL and HMI are started.
+-- 2) SubscribeVehicleData (speed, externalTemperature), OnVehicleData is allowed by policy for app_1 and app_2.
+-- 3) Navi App1 and navi app2 are registered and activated.
+-- Steps to reproduce:
+-- 1) Send SubscribeVehicleData (speed, externalTemperature) from app_1 => SDL responds SubscribeVehicleData
+-- ((speed:success),(externalTemperature:VEHICLE_DATA_NOT_AVAILABLE), SUCCESS).
```

Comment:
```suggestion
-- SDL should send "SUCCESS, (speed: SUCCESS), (externalTemperature: VEHICLE_DATA_NOT_AVAILABLE)" to mobile app1.
```
For consistency

Reply:
Hi @ShobhitAd.
Your comment was processed.